.github/workflows/build.yaml

```diff
@@ -37,10 +37,10 @@ jobs:
       - name: Build ISO
         run: |
           # Write out the channels file so it can be included
-          guix describe -f channels > channels.scm
+          guix time-machine -C './guix/base-channels.scm' -- describe -f channels > './guix/channels.scm'
```

Comment:
Now that we're doing this, maybe we should remove the channels configuration from the `Install Guix` step above?  Better to have the base channel list only in one place.

Reply:
Indeed, it's not needed at all now.

Can you confirm these is the only changes I need to perform? I have no way to test the syntax and the actual workflow:

```diff
24,35d23
<         with:
<           channels: |
<             (cons* (channel
<                     (name 'nonguix)
<                     (url "https://gitlab.com/nonguix/nonguix")
<                     ;; Enable signature verification:
<                     (introduction
<                      (make-channel-introduction
<                       "897c1a470da759236cc11798f4e0a5f7d4d59fbc"
<                       (openpgp-fingerprint
<                         "2A39 3FFF 68F4 EF7A 3D29  12AF 6F51 20A0 22FB B2D5"))))
<                     %default-channels)
```

Reply:
Yep, that's it!

Reply:
> Yep, that's it!

Changes have been commited.